integration-tests/utils/integration-helpers.js

```diff
@@ -72,9 +75,94 @@ const withUser = test => async args => {
   }
 }
 
+const addUserToBilling = withUserToken(async ({ email, token }) => {
+  const ajaxPage = await context.newPage()
+
+  await ajaxPage.goto(testUrl)
+  await signIntoTerra(ajaxPage, token)
+
+  await ajaxPage.evaluate((email, billingProject) => {
+    return window.Ajax().Billing.project(billingProject).addUser(['User'], email)
+  }, email, billingProject)
+  await ajaxPage.close()
+})
+
+const removeUserFromBilling = withUserToken(async ({ email, token }) => {
+  const ajaxPage = await context.newPage()
+
+  await ajaxPage.goto(testUrl)
+  await signIntoTerra(ajaxPage, token)
+
+  await ajaxPage.evaluate((email, billingProject) => {
+    return window.Ajax().Billing.project(billingProject).removeUser(['User'], email)
+  }, email, billingProject)
+  await ajaxPage.close()
+})
+
+const withBilling = test => async options => {
+  await addUserToBilling(options)
+
+  try {
+    await test({ ...options })
+  } finally {
+    await deleteCluster(options)
+    await removeUserFromBilling(options)
+  }
+}
+
+const trimClustersOldestFirst = _.flow(
+  _.remove({ status: 'Deleting' }),
+  _.sortBy('createdDate')
+)
+
+const currentCluster = _.flow(trimClustersOldestFirst, _.last)
+
+const getCurrentCluster = withUserToken(async ({ email, token }) => {
```

Comment:
Does this need to create its own page to make the call, or could we just inline it into `deleteCluster`?

Reply:
Let me double check, it was not working when I inlined it initially but I also made a few other changes which may have been the actual issue.